src/ApiAdapter.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/*
+ * This file is part of the Lakion package.
```

Comment:
I would vote for simply `This file is part of the Lakion sylius-api-php package.`, as it's a convention we use in other project's license files.